WalletWasabi/KeyManagement/KeyManager.cs

```diff
@@ -47,8 +47,11 @@ public class KeyManager
 		public string FilePath { get; private set; }
 		private object ToFileLock { get; }
 
+		[JsonProperty(Order = 5)]
```

Comment:
I think this should be 4 and HdPubKeys 5, because that's a huge array.

Reply:
I agree.

Reply:
I agree.

Reply:
I will change this and retest it.